homeassistant/components/statistics/sensor.py

```diff
@@ -203,6 +208,13 @@ def extra_state_attributes(self):
                 ATTR_CHANGE_RATE: self.change_rate,
             }
 
+        return {
+            ATTR_SAMPLING_SIZE: self._sampling_size,
```

Comment:
@MartinHjelmare I added this for similar parity; while I do find this one more handy it's just republishing an attribute that's part of the definition. If this entity were to move away from YAML like some of the other integrations then it would become handy again but right now it's a bit redundant (with the config). Should I remove this, too?

Reply:
Good question. Normally we don't add config to state attributes. But sometimes we do add state attributes that are not dynamic if they make automations viable and/or easier to write.

Reply:
In that case, I think we should leave it because it makes for handy template expressions (especially if you want to determine if you've counted to max value in an interval).